src/dlgIRC.cpp

```diff
@@ -1,6 +1,8 @@
 /***************************************************************************
+ *   Copyright (C) 2008-2017 The Communi Project                           *
  *   Copyright (C) 2008-2013 by Heiko Koehn - KoehnHeiko@googlemail.com    *
  *   Copyright (C) 2014 by Ahmed Charles - acharles@outlook.com            *
+ *   Copyright (C) 2017 by Fae - itsthefae@gmail.com                       *
```

Comment:
I would much prefer not to publish that information.  If it is required to validate and/or enforce the license terms, I would rather just remove the copyright lines.